securedrop/specialstrings.py

```diff
@@ -0,0 +1,93 @@
not getting up ever brown cats with pink ears or scamper, so curl up and sleep on the freshly laundered towels when in doubt, wash, yet why must they do that chew iPad power cord. Cats making all the muffins so you're just gonna scroll by without saying meowdy? yet adventure always. Chase imaginary bugs cat jumps and falls onto the couch purrs and wakes up in a new dimension filled with kitty litter meow meow yummy there is a bunch of cats hanging around eating catnip check cat door for ambush 10 times before coming in for attempt to leap between furniture but woefully miscalibrate and bellyflop onto the floor; what's your problem? i meant to do that now i shall wash myself intently yet burrow under covers. Fart in owners food ask to go outside and ask to come inside and ask to go outside and ask to come inside but groom forever, stretch tongue and leave it slightly out, blep sniff sniff. Human clearly uses close to one life a night no one naps that long so i revive by standing on chestawaken! nap all day mark territory, so stuff and things i'm bored inside, let me out i'm lonely outside, let me in i can't make up my mind whether to go in or out, guess i'll just stand partway in and partway out, contemplating the universe for half an hour how dare you nudge me with your foot?!?! leap into the air in greatest offense!. Kitty kitty sleep all day whilst slave is at work, play all night whilst slave is sleeping, plan your travel but purr as loud as possible, be the most annoying cat that you can, and, knock everything off the table lay on arms while you're using the keyboard. Sitting in a box pushes butt to face lies down and stare at imaginary bug. Love cat fur is the new black but roll on the floor purring your whiskers off. Howl uncontrollably for no reason dismember a mouse and then regurgitate parts of it on the family room floor for why must they do that eats owners hair then claws head while happily ignoring when being called trip owner up in kitchen i want food. There's a forty year old lady there let us feast lick the curtain just to be annoying yet pet my belly, you know you want to; seize the hand and shred it! so leave hair on owner's clothes immediately regret falling into bathtub walk on car leaving trail of paw prints on hood and windshield. Cat dog hate mouse eat string barf pillow no baths hate everything. Check cat door for ambush 10 times before coming in eat all the power cords for fish i must find my red catnip fishy fish sleep everywhere, but not in my bed throw down all the stuff in the kitchen where is my slave? I'm getting hungry lick the curtain just to be annoying. Kitty power meow to be let out lie on your belly and purr when you are asleep shake treat bag. Cat fur is the new black pet me pet me don't pet me i vomit in the bed in the middle of the night making bread on the bathrobe stare at ceiling light but wake up wander around the house making large amounts of noise jump on top of your human's bed and fall asleep again.
```

Comment:
Please find different copy for this.

Reply:
@creviera ^^ can we just choose another random text from anywhere else with same length?

Reply:
hah! yes please find more sensible copy, @ninavizz might have a good recommendation here

Reply:
This is dummy text I'd prepared a couple years ago, for client testing. What exactly are your concerns, @rmol?

https://github.com/freedomofpress/securedrop-ux/blob/master/dummydata.txt

Reply:
I'd simply like sample text that isn't embarrassing.

Reply:
agreed, cat ipsum and probably all lorem-ipsum alternatives text generators could produce an embarrassing  sequence of words. we can use the regular lorem ipsum generator (all latin) for the test strings where we really need ridiculously long messages.